services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+import init_django_orm  # noqa: F401
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]